### ISSUE TITLE
Crash when AutoForegound emits some failure cases while app in the background

### ISSUE THREAD

**Issue by hypest** [wordpress-mobile/WordPress-Android#7008]
```
Fatal Exception: org.greenrobot.eventbus.EventBusException: Invoking subscriber failed
       at org.greenrobot.eventbus.EventBus.handleSubscriberException(EventBus.java:505)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:487)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:479)
       at org.greenrobot.eventbus.HandlerPoster.handleMessage(HandlerPoster.java:67)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:145)
       at android.app.ActivityThread.main(ActivityThread.java:5951)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
Caused by java.lang.NullPointerException: Attempt to read from field 'android.os.Bundle android.app.Notification.extras' on a null object reference
       at android.support.v4.app.NotificationCompat.getExtras(NotificationCompat.java:4824)
       at android.support.v4.app.NotificationManagerCompat.useSideChannelForNotification(NotificationManagerCompat.java:283)
       at android.support.v4.app.NotificationManagerCompat.notify(NotificationManagerCompat.java:201)
       at android.support.v4.app.NotificationManagerCompat.notify(NotificationManagerCompat.java:191)
       at org.wordpress.android.util.AutoForeground.notifyState(AutoForeground.java:162)
       at org.wordpress.android.ui.accounts.login.LoginWpcomService.setState(LoginWpcomService.java:184)
       at org.wordpress.android.ui.accounts.login.LoginWpcomService.onAccountChanged(LoginWpcomService.java:345)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:485)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:479)
       at org.greenrobot.eventbus.HandlerPoster.handleMessage(HandlerPoster.java:67)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:145)
       at android.app.ActivityThread.main(ActivityThread.java:5951)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
```

**Comment by hypest** [comment]
As @mzorz has pinpointed, seems to happen when the `FAILURE_FETCHING_ACCOUNT` LoginPhase gets emitted.

**Comment by khaykov** [comment]
Fixed in #7009
